pages/docs/balance/open-banking/plaid.mdx

```diff
@@ -40,25 +40,55 @@ Use the following credentials to simulate successful authentication with test ba
 - **Username**: `user_good`
 - **Password**: `pass_good`
 
+### Returning User Testing in Sandbox
+
+Plaid's Sandbox includes pre-seeded test users for validating different returning user scenarios. Use the phone numbers and OTP (always `123456`) below to simulate these cases:
+
+| Scenario                                     | Phone Number |
+| -------------------------------------------- | ------------ |
+| New User                                     | 415-555-0010 |
+| Verified Returning User                      | 415-555-0011 |
+| Verified Returning User: Linked New Account  | 415-555-0012 |
+| Verified Returning User: Linked OAuth Bank   | 415-555-0013 |
+| Verified Returning User + New Device         | 415-555-0014 |
+| Verified Returning User: Auto Account Select | 415-555-0015 |
+
 ### Key Notes About Plaid Test Banks
 
 Plaid offers two test banks: one for **checking** accounts and one for **savings** accounts. Both have predefined account and routing numbers. This means:
 
 - You can add up to two banks per customer (one checking, one savings).
```

Comment:
@ShreyaThapa Could we capitalize and add a <ContentTooltip> Around `customer`? Thinking through how we can start integrating these more throughout the docs 😄

Reply:
Sure thing! Did you mean adding tildes around Customer, like this - `Customer`?

Reply:
Weird, ContentTooltip didn't show up in my comment. Not tildes. I'm thinking of using the component we built for content tooltips